packages/scss/src/components/_header.scss

```diff
@@ -9,3 +9,12 @@
 	position: relative;
 	z-index: 1;
 }
+
+.header-left {
+	display: flex;
+}
+
+.header-right {
+	display: flex;
+	margin-left: auto;
```

Comment:
If we go this path, you should remove the "justify-content: space-between" from the `header` class

Reply:
woops :)